config/environments/test.rb

```diff
@@ -41,7 +41,10 @@
   config.action_mailer.delivery_method = :test
 
   # Print deprecation notices to the stderr.
-  config.active_support.deprecation = :stderr
+  # config.active_support.deprecation = :stderr
+
+  # send deprecation notices to a log file
+  config.active_support.deprecation = :log
```

Comment:
clean up the console outputs for deprecation warnings and push them to a log file